### ISSUE TITLE
[feature request] Support SMB share

### ISSUE THREAD

**Issue by J4gQBqqR** [chr56/Phonograph_Plus#2]

**Comment by chr56** [comment]
What do you mean? Music file sharing of Phonograph is through Android system (In short, Android provides the method to share files),and I (or kabouzeid) have not written any codes of actual "sharing", so it is Android's duty to provide &r control file sharing. Besides, what does "encrypted" you said mean? Please provide more details. I'm confused.🤔

**Comment by J4gQBqqR** [comment]
Some music player and video plyer, such as UAPP and VLC support accessing multimedia files stored on network. These multimedia files are normally offerred by Samba share or NFS share.

Encrypted means in Samba configuration, admin cab configure the network share to transmit files via predefined encrypted standard. Samba will transmit the files encrypted, the client who is reading the file will decrypt it.

**Comment by chr56** [comment]
Well, if you mean accessing remote multimedia files via SMB,NFS and etc. , I am afraid that I can not help you.
Phonograph accesses files via MediaStore (provided by Android OS), which can only provided local files access (it seems). And Phonograph isn't a file explorer or remote files access manager (at least so far), though even some remote files managers provide access for other apps, I am NOT planning to add features of access remote files (including add a remote file client in Phonograph or use available existing interfaces provided by other remote files managers installed on the phone).
In a nutshell, Phonograph Plus is just a plain LOCAL music player, not a file explorer. Sorry, I can't help you.

**Comment by J4gQBqqR** [comment]
Thank you for explaining the scope of the ptoject.
